addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -234,10 +240,20 @@ const LinkPopoverWidget = Widget.extend({
      */
     _onEditLinkClick(ev) {
         ev.preventDefault();
-        this.options.wysiwyg.toggleLinkTools({
-            forceOpen: true,
-            link: this.$target[0],
-        });
+        const content = this.$target[0].children[0];
+        if (isImg(content)) {
+            const optionsDocument = this.options.wysiwyg.el.ownerDocument;
+            const urlFieldEl = optionsDocument.querySelector('[data-name="media_url_opt"] input');
+            if (urlFieldEl) {
+                urlFieldEl.focus();
+                urlFieldEl.select();
+            }
+        } else {
```

Comment:
- We already have a not-so-pretty `activate_image_link_tool` event which can do this, maybe use it?
- Does checking the first children enough? What happens currenly if we have a button with a mini-picture icon and a text? If you click the button text (not the image), I guess the normal link tools (not the image dedicated ones) are available and you should call toggleLinkTools?

Reply:
- the event works fine indeed
- for the `isImg()` check, I now use the wysiwyg's `lastElement` - from `$target` which is the `<a>` I cannot determine if it is the text or the image that is selected in a button with an icon